### ISSUE TITLE
some confusion about libtorch-BatchNorm2d

### ISSUE THREAD

**Issue by xiaose26** [pytorch/pytorch#65809]
## envs
* libtorch 1.9.1
* ubuntu 20.04
* RTX2060 6G
## code
```C++
// define model
// .h
struct TestM:torch::nn::Module{
private:
     torch::nn::Conv2d      conv1{nullptr};
    torch::nn::BatchNorm2d bn1{nullptr};
public:
    TestM();
    torch::Tensor forward(torch::Tensor x);
};
//.cpp
TestM::TestM()
{
    conv1    = torch::nn::Conv2d(conv_options(3, 64, 7, 2, 3, 1, 1, false));
    bn1      = torch::nn::BatchNorm2d(torch::nn::BatchNorm2dOptions(64).affine(true).track_running_stats(true));
   register_module("conv1", conv1);
    register_module("bn1", bn1);
}

//load weights
torch::autograd::GradMode::set_enabled(false);
std::shared_ptr<TestM> model= make_shared<TestM>(TestM);
    for(auto b : model->named_buffers(true))
    {
        b.value().copy_(weights[b.key()]);
    }
    for(auto m : model->named_parameters())
    {
        m.value().copy_(weights[m.key()]);
        // cout << m.key() << endl;
    }
    for(auto mm : model->modules())
    {
        mm->eval();
    }

// <-------------------------------------------------here get differnet results(x1,x2,x3)
torch::Tensor TestM::forward(torch::Tensor x)
{
    x=conv1->forward(x);
    at::Tensor x1=bn1->forward(x);  
    at::Tensor x2= F::batch_norm( x, bn1->running_mean, bn1->running_var,
        F::BatchNormFuncOptions().training(false).eps(1e-05).momentum(0.1).weight(bn1->weight).bias(bn1->bias));
    at::Tensor x3= x.sub(bn1->running_mean.reshape({-1, 1, 1}))
           .div(torch::pow(bn1->running_var.reshape({-1, 1, 1}) + 1e-05, 0.5))
           .mul(bn1->weight.reshape({-1, 1, 1}))
           .add(bn1->bias.reshape({-1, 1, 1}));
```
result `x1=x2!=x3` ,if want `bn1->forward` run as `x3` ,how should i set the parameters of `torch::nn::BatchNorm2dOptions`? Thanks!
add : some GPU is ok :sob:

**Comment by saketh-are** [comment]
Hey @xiaose26, we use github issues only for bug reports or feature requests. Please use the forum to ask questions: https://discuss.pytorch.org/
